account_asset_management_xlsx/report/report_account_asset_xlsx.py

```diff
@@ -0,0 +1,459 @@
+        fm_title_table = fm_default.copy()
+        fm_title_table.update({"bold": True, "align": "center"})
+        c_fm_title_table = self._create_format(workbook, fm_title_table)
+        fm_values.update({"fm_title_table": c_fm_title_table})
+
+        # ---------------------------------------------------------------------
+        # Table format
+        # ---------------------------------------------------------------------
+        fm_table = fm_default.copy()
+        fm_table.update(
+            {
+                "font_size": 11,
+                "bold": True,
+                "align": "vcenter",
+                "bg_color": "#0070c0",
+                "font_color": "#ffffff",
+            }
+        )
+        c_fm_table = self._create_format(workbook, fm_table)
+        fm_values.update({"fm_table": c_fm_table})
+        # ---------------------------------------------------------------------
+        fm_table_header = fm_table.copy()
+        fm_table_header.update(
+            {
+                "font_size": 10,
+                "bg_color": "#d9d9d9",
+                "font_color": "#000000",
+            }
+        )
+        c_fm_table_header = self._create_format(workbook, fm_table_header)
+        fm_values.update({"fm_table_header": c_fm_table_header})
+        # ---------------------------------------------------------------------
+        fm_table_bold = fm_table.copy()
+        fm_table_bold.update(
+            {
+                "font_size": 10,
+                "bg_color": "#d9d9d9",
+                "font_color": "#000000",
+            }
+        )
+        c_fm_table_bold = self._create_format(workbook, fm_table_bold)
+        fm_values.update({"fm_table_bold": c_fm_table_bold})
+        # ---------------------------------------------------------------------
+        fm_table_number = fm_table.copy()
+        fm_table_number.update(
+            {
+                "font_size": 10,
+                "bold": False,
+                "num_format": "#,##0.00",
+            }
+        )
+        c_fm_table_number = self._create_format(workbook, fm_table_number)
+        fm_values.update({"fm_table_number": c_fm_table_number})
+        # ---------------------------------------------------------------------
+        fm_table_number_bold = fm_table.copy()
+        fm_table_number_bold.update(
+            {
+                "font_size": 10,
+                "num_format": "#,##0.00",
+                "bg_color": "#d9d9d9",
+                "font_color": "#000000",
+            }
+        )
+        c_fm_table_number_bold = self._create_format(workbook, fm_table_number_bold)
+        fm_values.update({"fm_table_number_bold": c_fm_table_number_bold})
+        # ---------------------------------------------------------------------
+        fm_table_date = fm_default.copy()
+        fm_table_date.update(
+            {
+                "num_format": "dd/mm/yyyy",
```

Comment:
date format get from date_format of user language.
search for this to have a look:
`date_format = posix_to_ldml(lang.date_format, locale=locale)`